go.mod

```diff
@@ -24,7 +24,6 @@ require (
 	github.com/joho/godotenv v1.2.0
 	github.com/json-iterator/go v1.1.8 // indirect
 	github.com/lib/pq v0.0.0-20180123210206-19c8e9ad0095
-	github.com/modern-go/reflect2 v1.0.1 // indirect
```

Comment:
Any idea why these deps got removed?

Reply:
Hmmm... I don't believe that they should be getting removed. The reflect2 dependency should be there because of the `github.com/json-iterator/go v1.1.8 // indirect` that shows up 2 lines earlier. Coincidentally, this dependency is the one I used as an example for 'git mod why ...' command for the Go Mod section in our Go style guide: https://github.com/conjurinc/docs/pull/139/files#diff-b48d7f4cab90a224552e7e2f355e0a71R329-R337

Reply:
My best guess is that these deletions were due to some cruft leftover in my local workspace. To check this, I undid those deletions and forced a build (which should update go.mod as necessary), and those entries stayed in go.mod (as they should). Change set has been updated.

Reply:
I'm still seeing them as deleted, but your comment implies this file should not have changed in this PR - can you please clarify?

Reply:
After rebase with latest master, this issue goes away.

I think the go command had been "doing the right thing" in removing these dependencies from go mod. In fact, in the latest master, these have actually been deleted. I suspect that someone may have done a prior merge of changes with a go.mod that wasn't quite up-to-date. This could happen, I suppose, if someone does a merge in a workspace where they haven't executed any go commands, or they did 'git checkout go.mod' for some reason before committing. Either way, the go command was trying to update go.mod to what it should be.